### PR TITLE
Add Meta field to Attachment

### PR DESCRIPTION
**Pull request by pacmessica**

### CODE REVIEW

File: mastodon.go [hunk 1]
```diff
@@ -225,13 +225,28 @@ type History struct {
...
+}
+
+// AttachmentMeta holds information for attachment metadata.
+type AttachmentMeta struct {
+	Original Size `json:"original"`
+	Small    Size `json:"small"`
+}
+
+// Size holds information for attatchment size.
+type Size struct {
```
Comment (mattn) [inline mastodon.go]: please rename type Size to AttachmentSize.

Comment (pacmessica) [reply mastodon.go]: done :)

Comment (pacmessica) [reply mastodon.go]: @mattn

Comment (mattn) [reply mastodon.go]: Oops Sorry... Since I looked your diff in few days ago, the notification has gone at my GitHub notifications. LGTM.

**Pull request merged**